### PR TITLE
Query and update animals table

### PR DESCRIPTION
**Pull request by yin-ka**
**In this PR**,
- I added a column 'species' into the table
- I inserted new data and added my insertions into the data.sql file.
- Inside a transaction, I updated the table by setting the species column to unspecified and rollbacked the changes.
- Inside a transaction, I updated the table by setting the species column to 'digimon' for all animals that have a name ending - in 'mon', and to 'pokemon' for all animals that don't have species already set, and I committed the changes.
- Below are the screenshots:
Add new column
Insert more values into table
set species to unspecified
RollBack
Colomn species after first transaction
After first commit
verifying if all records in the animals table still exists
after second comit
How many animals are there?
How many animals have never tried to escape?
What is the average weight of animals?
Who escapes the most, neutered or not neutered animals?
What is the minimum and maximum weight of each type of animal?
What is the average number of escape attempts per animal type of those born between 1990 and 2000?

### CODE REVIEW

**Comment by addod19** [comment]
## Hi @yin-ka, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because the README file lacks relevant headers like the `Prerequisites`, `Usage`, etc

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: data.sql [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+ date_of_birth, escape_attempts,
+  neutered, weight_kg) VALUES
+   ('Devimon', '2017-05-12',
+    5, true, 11.00);
+
+-- INSERT DAY TWO
+INSERT INTO animals(name,
+ date_of_birth, escape_attempts,
+  neutered, weight_kg) VALUES('Charmander',
+   '2020-02-08', 0, false, 11.00),
```
Comment (Ghiftee) [inline data.sql]: - [ ] Good job adding these values, however, you are missing a tiny detail. The weight_kg of Charmander is supposed to be a negative value as shown in the requirement below. Please update it.
```Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg```

File: queries.sql [hunk 2]
```diff
@@ -14,3 +14,61 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
...
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.40 AND 17.30;
 
+-- Update and Delete changes to species
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
```
Comment (Ghiftee) [inline queries.sql:24]: - [ ] After rollback, please verify that the changes were undone e.g 
```
ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @yin-ka 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ Queries are successful
✔️ Screenshots are added
✔️ README is professional

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review 👇🏽 _

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- Kindly consider adding the queries for each transaction above their screenshots for easier identification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: data.sql (hunk 1 above)
Comment (yin-ka) [reply data.sql]: Okay, thanks

File: queries.sql (hunk 2 above)
Comment (yin-ka) [reply queries.sql:24]: Thank you. Changes made.

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**